src/variableinspector.ts

```diff
@@ -250,48 +291,62 @@ export class VariableInspectorPanel
       }
 
       // Add delete icon and onclick event
-      let cell = row.insertCell(0);
+      let cell = document.createElement('jp-data-grid-cell') as DataGridCell;
       cell.title = 'Delete Variable';
       cell.className = 'jp-VarInspector-deleteButton';
+      cell.gridColumn = '1';
       const ico = closeIcon.element();
+      ico.className = 'icon-button';
       ico.onclick = (ev: MouseEvent): any => {
-        this.source?.performDelete(name);
+        this.removeRow(name);
       };
       cell.append(ico);
+      row.appendChild(cell);
 
       // Add onclick event for inspection
-      cell = row.insertCell(1);
+      cell = document.createElement('jp-data-grid-cell') as DataGridCell;
       if (item.isMatrix) {
         cell.title = 'View Contents';
         cell.className = 'jp-VarInspector-inspectButton';
         const ico = searchIcon.element();
```

Comment:
Would you mind using a `jp-button` to wrap the icon in it (you can use the appearance _stealth_)?